torchgeo/datasets/utils.py

```diff
@@ -27,26 +27,8 @@
 from torchvision.datasets.utils import check_integrity, download_url
 from torchvision.utils import draw_segmentation_masks
 
-__all__ = (
-    'check_integrity',
-    'DatasetNotFoundError',
-    'RGBBandsMissingError',
-    'download_url',
-    'download_and_extract_archive',
-    'extract_archive',
-    'BoundingBox',
-    'disambiguate_timestamp',
-    'working_dir',
-    'stack_samples',
-    'concat_samples',
-    'merge_samples',
-    'unbind_samples',
-    'rasterio_loader',
-    'sort_sentinel2_bands',
-    'draw_semantic_segmentation_masks',
-    'rgb_to_mask',
-    'percentile_normalization',
-)
+# Only include import redirects
+__all__ = ('check_integrity', 'download_url')
```

Comment:
`torchgeo.datasets.check_integrity` is the same as `torchvision.datasets.check_integrity`, but we alias it here for consistency. We don't really need to do this. But for now, `__all__` is needed to avoid unused import errors.